bower.json

```diff
@@ -69,5 +69,9 @@
         "rangy-selectionsaverestore.js"
       ]
     }
+  },
+  "resolutions": {
+    "ember": "beta",
```

Comment:
Should the ember version be pegged [above](https://github.com/Addepar/ember-widgets/pull/178/files#diff-0a08a7565aba4405282251491979bb6bL27) to `~1.10.0` instead of using beta?